Dockerfile

```diff
@@ -2,6 +2,7 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 AS builder
 
 WORKDIR /workspace
+ARG GOFLAGS
```

Comment:
Note: I think a downstream change will be needed to incorporate the GOFLAGS into the build
@beraldoleal @jensfr

Reply:
removed